blockchain/pool.go

```diff
@@ -252,7 +252,8 @@ func (pool *BlockPool) AddBlock(peerID p2p.ID, block *types.Block, blockSize int
 			peer.decrPending(blockSize)
 		}
 	} else {
-		// Bad peer?
```

Comment:
@milosevic , peerID is use to make sure here is an Bad peer. 
previously， normal logic can meet here， because redo was send message to chan，and deal it delayed，so  redo may apply to an good peer.